{
    "error": "HTTPSConnectionPool(host='cxsecurity.com', port=443): Max retries exceeded with url: /issue/WLB-2017090094 (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x7f01d40bfcd0>, 'Connection to cxsecurity.com timed out. (connect timeout=10)'))"
}